natgeo/mortar
# [SYTO-753] A couple of small design tweaks to POD promo

These changes address a couple of things from design feedback 

To test:

- Pull bugfixes/SYTO-753
- Verify unit tests pass by running npm test
- Run gulp serve
- Verify component at http://localhost:3000/components/podpromo/index.html using the configurator
- Verify component also renders as expected at http://localhost:3000/components/
